ReserveSpot.Services.Infrastructure.Build/Program.cs

```diff
@@ -51,7 +51,7 @@ static void Main(string[] args)
 
                                 TargetDotNetVersion = new TargetDotNetVersion
                                 {
-                                    DotNetVersion = "6.0.100",
+                                    DotNetVersion = "7.0.0-preview.1",
```

Comment:
Please do not do this here.

Reply:
Reverted to 6.0.100 . I will do that in a separate PR